Geometry/CSCGeometryBuilder/plugins/CSCGeometryValidate.cc

```diff
@@ -0,0 +1,358 @@
+/*
+//\class CSCGeometryValidate
+
+ Description: CSC GeometryValidate from DD & DD4hep
+
+//
+// Author:  Sergio Lo Meo (sergio.lo.meo@cern.ch) following what Ianna Osburne made for DTs (DD4HEP migration)
+//          Created:  Thu, 05 March 2020 
+*/
+
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "Geometry/CSCGeometry/interface/CSCGeometry.h"
+#include "Geometry/CSCGeometry/interface/CSCChamber.h"
+#include "Geometry/CSCGeometry/interface/CSCLayer.h"
+#include "Geometry/CSCGeometry/interface/CSCLayerGeometry.h"
+#include <Geometry/CSCGeometry/interface/CSCStripTopology.h>
+#include <Geometry/CSCGeometry/interface/CSCWireTopology.h>
+#include "Geometry/CommonTopologies/interface/StripTopology.h"
+#include "Geometry/Records/interface/MuonGeometryRecord.h"
+
+#include "Fireworks/Core/interface/FWGeometry.h"
+
+#include "DataFormats/GeometrySurface/interface/RectangularPlaneBounds.h"
+#include "DataFormats/GeometrySurface/interface/TrapezoidalPlaneBounds.h"
+
+#include <TFile.h>
+#include <TH1.h>
+
+#include <limits>
+#include <string>
+#include <type_traits>
+#include <algorithm>
+
+using namespace std;
+
+template <class T>
```

Comment:
this is not used here, please, delete from line #42 to line #49

Reply:
They are necessary for: "if (almost_equal(val1, val2, tolerance_))" and "name.c_str()" (in THD1)